src/scheduler/worker.rs

```diff
@@ -19,16 +19,25 @@ pub struct GCWorkerShared<VM: VMBinding> {
     pub parked: AtomicBool,
     /// Worker-local statistics data.
     stat: AtomicRefCell<WorkerLocalStat<VM>>,
-    /// Incoming work packets to be executed by the current worker.
-    pub local_work_bucket: WorkBucket<VM>,
+    /// A queue of GCWork that can only be processed by the owned thread.
+    pub local_work: ArrayQueue<Box<dyn GCWork<VM>>>,
+    /// Local work packet queue.
+    pub local_work_buffer: Worker<Box<dyn GCWork<VM>>>,
```

Comment:
This is not actually shared. We don't expect any other thread to access this. The `Stealer` could be shared so that other threads can steal, but the `Worker` should not be shared.

Reply:
Yes. `local_work_buffer` is moved to `GCWorker`.